src/scripts/generate_docs.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+from dataclasses import dataclass
+from codemodder.registry import load_registered_codemods
+from pathlib import Path
+
+
+@dataclass
+class DocMetadata:
+    """Codemod specific metadata only for documentation"""
+
+    importance: str
+    guidance_explained: str
+    need_sarif: str = "No"
+
+
+# codemod-specific metadata that's used only for docs, not for codemod API
```

Comment:
I'm not totally in love with this as a dict but neither do I care for having a separate json file

Reply:
I feel like those could go as a dataclass field within the codemods themselves.

Reply:
I don't disagree but Dan wanted to keep this totally separate from the codemods. I don't mind either way at this time.